### PR TITLE
Spaces

### PR DESCRIPTION
**Pull request by dapatil211**

### CODE REVIEW

File: hive/agents/rainbow.py [hunk 1]
```diff
@@ -18,17 +18,17 @@
 from hive.replays.replay_buffer import BaseReplayBuffer
 from hive.utils.loggers import Logger
 from hive.utils.schedule import Schedule
-from hive.utils.utils import LossFn, OptimizerFn, seeder
```
Comment (alirahkay) [inline hive/agents/rainbow.py:21]: Please only include related changes in the PR.

File: hive/utils/registry.py [hunk 2]
```diff
@@ -202,7 +202,7 @@ def get_parsed_args(arguments, prefix=None):
     """Helper function that takes a dictionary mapping argument names to types, and
     extracts command line arguments for those arguments. If the dictionary contains
     a key-value pair "bar": int, and the prefix passed is "foo", this function will
-    look for a command line argument "\-\-foo.bar". If present, it will cast it to an
+    look for a command line argument "--foo.bar". If present, it will cast it to an
```
Comment (alirahkay) [inline hive/utils/registry.py:205]: Non-related change.

**Review by alirahkay — COMMENTED** [review]
Overall LGTM!
Thanks for updating the tests.
The last thing to check is if all the config files can be run with these new changes. Did you check them too?

File: hive/agents/rainbow.py [hunk 3]
```diff
@@ -18,17 +18,17 @@
...
+from hive.utils.utils import LossFn, OptimizerFn
 
 
 class RainbowDQNAgent(DQNAgent):
     """An agent implementing the Rainbow algorithm."""
```
Comment (hnekoeiq) [inline hive/agents/rainbow.py:26]: I believe the `legal_moves_rainbow` agent does not need any update. But could you also confirm that?

**Review by hnekoeiq — APPROVED** [review]
LGTM!

**Review by alirahkay — APPROVED** [review]
I talked to Darshan on Slack, and he confirmed that the config files should not be affected by this PR. Hopefully, no change is needed. 
LGTM!

File: hive/agents/rainbow.py (hunk 3 above)
Comment (dapatil211) [reply hive/agents/rainbow.py:26]: Nope, should be good as is

**Pull request merged**